app/views/shared/_jobseeker_message.html.slim

```diff
@@ -3,6 +3,4 @@
     h3.govuk-summary-card__title
       = jobseeker_message_display_name(message, job_application)
   .govuk-summary-card__content
-    p.govuk-body class="govuk-!-font-weight-bold" = "Date: #{message.created_at.to_fs(:date_at_time)}"
-    p.govuk-body class="govuk-!-font-weight-bold" = message.conversation.title
-    p.govuk-body class="message-content" = simple_format(message.content)
+    p.govuk-body class="message-content" = message.content
```

Comment:
Just a concern that may be overzealous: We need to be sure that the message input form is being sanitised when the user introduces it. Given that this change removed the `simple_format` sanitisacion.